### PR TITLE
Gangams/release changes

### PR DESCRIPTION
**Pull request by ganga1980**
changes related to release ciprod08052020.

### CODE REVIEW

File: ReleaseNotes.md [hunk 1]
```diff
@@ -11,6 +11,18 @@ additional questions or comments.
 
 Note : The agent version(s) below has dates (ciprod<mmddyyyy>), which indicate the agent build dates (not release dates)
 
+### 08/05/2020 -
+##### Version microsoft/oms:ciprod07152020 Version mcr.microsoft.com/azuremonitor/containerinsights/ciprod:ciprod08052020 (linux)
```
Comment (vishiy) [inline ReleaseNotes.md:15]: incorrect versions here (both win & lin)

Comment (ganga1980) [reply ReleaseNotes.md]: > incorrect versions here (both win & lin)

> fixed. thanks for catching.

File: ReleaseNotes.md [hunk 2]
```diff
@@ -11,6 +11,18 @@ additional questions or comments.
...
+##### Version microsoft/oms:win-ciprod05262020-2 Version mcr.microsoft.com/azuremonitor/containerinsights/ciprod:win-ciprod08052020 (windows)
+##### Code change log
```
Comment (vishiy) [inline ReleaseNotes.md:17]: few more -
* Bug fix for showing node as 'not ready' when there is disk pressure
* oneagent integration (disabled by default)
* Add region check before sending alertable metrics to MDM
* Telemetry fix for agent telemetry for sov. clouds

File: build/version [hunk 3]
```diff
@@ -5,8 +5,8 @@
 CONTAINER_BUILDVERSION_MAJOR=10
 CONTAINER_BUILDVERSION_MINOR=0
 CONTAINER_BUILDVERSION_PATCH=0
-CONTAINER_BUILDVERSION_BUILDNR=1
-CONTAINER_BUILDVERSION_DATE=20200526
+CONTAINER_BUILDVERSION_BUILDNR=4
```
Comment (vishiy) [inline build/version:8]: why bump from 1 to 4 ? due to 2 hotfixes ?

File: ReleaseProcess.md [hunk 4]
```diff
@@ -27,7 +33,9 @@ Image automatically synched to MCR CN from Public cloud MCR.
 
 ## AKS
 
-Make PR against [AKS-RP](https://msazure.visualstudio.com/CloudNativeCompute/_git/aks-rp?version=GBmaster) repo with chart update(s)
+- Make PR to prebake the latest image (https://github.com/Azure/AgentBaker)
+- Make PR against [AKS-RP](https://msazure.visualstudio.com/CloudNativeCompute/_git/aks-rp?version=GBmaster) repo with chart update(s)
```
Comment (vishiy) [inline ReleaseProcess.md:37]: This is public facing. We should probably not doc how we deploy to AKS here.

File: build/version (hunk 3 above)
Comment (ganga1980) [reply build/version:8]: > why bump from 1 to 4 ? due to 2 hotfixes ?

> That's right.

File: ReleaseProcess.md (hunk 4 above)
Comment (ganga1980) [reply ReleaseProcess.md:37]: > This is public facing. We should probably not doc how we deploy to AKS here.

Make sense.  Let me remove these and we can add it to our internal wiki

File: ReleaseNotes.md (hunk 2 above)
Comment (ganga1980) [reply ReleaseNotes.md:17]: > * Bug fix for showing node as 'not ready' when there is disk pressure
> * oneagent integration (disabled by default)
> * Add region check before sending alertable metrics to MDM
> * Telemetry fix for agent telemetry for sov. clouds

> Fixed.

File: ReleaseProcess.md (hunk 4 above)
Comment (ganga1980) [reply ReleaseProcess.md]: > > This is public facing. We should probably not doc how we deploy to AKS here.
> 
> Make sense. Let me remove these and we can add it to our internal wiki

> removed.

**Pull request merged**